src/workloads/execution/UnshardedLookup.yml

```diff
@@ -71,6 +95,38 @@ Actors:
             }}
           ]
         cursor: {batchSize: *batchSize}
+    - OperationMetricsName: MatchAllShardsLookupUnsharded
+      OperationName: RunCommand
+      OperationCommand:
+        aggregate: Collection2
+        pipeline:
+          [
+            {$match: {
+              c: {$in: [84, 112, 253, 396, 495, 515, 638, 754, 872, 998]}
+            }},
+            {$lookup: {
+              from: "Collection1",
+              let: {b0: "$b"},
+              pipeline: [{$match: {$expr: {$eq: ["$$b0", "$b"]}}}],
+              as: "joined"
+            }}
+          ]
+        cursor: {batchSize: *batchSize}
+    - OperationMetricsName: MatchSingleShardLookupUnsharded
+      OperationName: RunCommand
+      OperationCommand:
+        aggregate: Collection2
+        pipeline:
+          [
+            {$match: {a: 23}},
```

Comment:
At first glance I was worried that there might be cases where we wouldn't get any matches at all, but looks like that will only happen `0.99^1000 = 0.004%` of the time. Nice!

Reply:
Yeah, although it would be nice if we had something like `{^SequentialInt: {min: 1, max: 100}}` for this purpose.

Reply:
Agreed! Militsa edited the `MonotonicLoader` actor to accomplish something similar for the first iteration of her [$graphLookup benchmarks](https://github.com/mongodb/genny/pull/506), but as far as I know there's currently no built-in way in Genny to do this. Sounds like a feature request!

Reply:
Agreed! Militsa edited the MonotonicLoader actor to accomplish something similar for the first iteration of her [$graphLookup benchmarks](https://github.com/mongodb/genny/pull/506), but as far as I know there's currently no built-in way in Genny to do this. Sounds like a feature request!